client/src/store/Location.ts

```diff
@@ -0,0 +1,17 @@
+import { GirderModel } from '@/lib/api/viame.service';
+
+export interface LocationState {
+    location: null | GirderModel;
+}
+
+export default {
+  namespaced: true,
+  state: {
+    location: null,
+  } as {location: null | GirderModel},
+  mutations: {
+    setLocation(state: { location: null | GirderModel}, location: GirderModel) {
```

Comment:
This isn't what you want to do here.  Should let Vuex's type definitions and a module declaration figure out the state typing.

Check out https://github.com/VIAME/VIAME-Web/pull/172